packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -194,6 +201,16 @@ export class ProjectService {
         const savedProject = await this.projectModel.getWithSensitiveFields(
             projectUuid,
         );
+        if (
+            user.ability.cannot(
+                'update',
+                subject('Project', {
+                    organizationUuid: savedProject.organizationUuid,
+                }),
+            )
+        ) {
+            throw new ForbiddenError();
```

Comment:
Should we create the job after checking the permission ? otherwise we need to mark the job as failed.